viahtml/templates/banner.html

```diff
@@ -111,5 +111,7 @@
         var embed_script = document.createElement("script");
         embed_script.src = "{{ h_embed_url }}";
         document.head.appendChild(embed_script);
+
+        setupExternalLinkHandler({{external_link_mode(env) | tojson}});
```

Comment:
I'd copy pasted the code that actually does the work for this above from Via ages ago. We just need to turn it on.

The `tojson` here will convert our python string into a well formatted javascript string, and prevents any nonsense from leaking out.